### DISCUSSION TITLE
Using context menu outside finder (🦆 - macOS)

### DISCUSSION THREAD

**Discussion by Cassian-Andor** [iterate-ch/cyberduck#14502] [Q&A]
Is it possible to use the context menu (right click) that shows up in Finder.app when in a mounted drive, in other file managers? For example Path Finder, ForkLift, QSpace, Marta. None of them show the `Mountain Duck` context menu when right clicking on a MD volume (useful for unlocking or locking a vault for example).

Is this a limitation of how the context menu is implemented in macOS? If so where can I read more info to maybe try a hack (like some sort of command line that I can append to a custom action in one of the file managers that works the same as in finder).

I do not even know where to start to look at on Apple's documentation.

**Comment by tatze96** [comment]
The Mountain Duck context menu or, more specifically, the Mountain Duck Finder Extension can be used and accessed within Finder replacements, given that they have support for Finder Extensions.

Please get in touch with the support of your preferred Finder replacement or check their documentation if you are unsure whether or not they have Finder Extension support.
